src/actions/index.js

```diff
@@ -78,12 +78,27 @@ export function fetchSectionComments(hearingId, sectionId) {
   };
 }
 
-export function postSectionComment(hearingId, sectionId, text, authorName, pluginData = null, authCode = "") {
+export function postSectionComment(hearingId,
```

Comment:
Will do. This resulted into miscalls way back when there were less arguments involved.